src/simplay-web/test/event/StoreSetCapacityEvent.test.ts

```diff
@@ -75,12 +75,12 @@ describe('StoreSetCapacityEvent tests', function () {
     const spooler = new SimulationSpooler(simData, container);
     await new Promise((resolve) => setTimeout(resolve, 100));
 
-    // set the content to [] to make the text predictable
+    // set the content to "" to make the text predictable
```

Comment:
what do you mean by predictable?

Reply:
Since this test is testing the capacity text generation, I dont want any text generated from the content. But the wording isn't clever, I agree that any other text is just as predictable ;)